packages/devtools_app/lib/src/inspector/inspector_tree_flutter.dart

```diff
@@ -591,5 +613,52 @@ class InspectorRowContent extends StatelessWidget {
         ),
       ),
     );
+
+    // Wrap with error tooltip/marker if there is an error for this node's widget.
+    return errorText == null
+        ? rowWidget
+        : Stack(
+            children: [
+              rowWidget,
+              ErrorIndicator(
+                errorText: errorText,
+                indent: currentX - rowHeight * 2,
```

Comment:
why are we subtracting `rowHeight * 2`? if this is for the width of the error dot, instead subtract some const for that specifically instead of repurposing the `rowHeight` const. You could add some const to the ErrorIndicator class `leadingOffset` or similar.

Reply:
It's to account for the other widgets (expand/collapse and guide lines). I've added a `static const double indicatorOffset = -(defaultIconSize * 2 + denseSpacing);` for this and a comment explaining.